freegs4e/critical.py

```diff
@@ -675,6 +681,46 @@ def inside_mask(
     return mask
 
 
+def inside_mask(
+    R,
+    Z,
+    psi,
+    opoint,
+    xpoint=[],
+    mask_outside_limiter=None,
+    psi_bndry=None,
+    use_geom=True,
+):
```

Comment:
Could a docstring be added? It can refer to `inside_mask_` but should also explain what `use_geom` does

Reply:
Resolved in #12